lessons/Projects/project-beautiful-data/langs/en-us/index.adoc

```diff
@@ -0,0 +1,99 @@
+Students advance their programming skills by using code to add their own flair and style to data that matters to them.
+
+=== Launch
+
+Data visualization is where data science and creativity mix. Data displays are meant to __tell a story__, and storytelling is an art form!
+
+@lesson-instruction{
+Take a moment to explore the resources below, and choose **one display** that really interests you:
+
+- @opt-online-exercise{https://pudding.cool/projects/vocabulary/index.html, The Largest Vocabulary in Hip Hop}
+- @opt-online-exercise{https://insightfulinteraction.com/immigration200years.html, 200 Years of Immigration}
+- @opt-online-exercise{https://informationisbeautiful.net/, Information is Beautiful}
+}
+
+@teacher{Allow students ample time to check out the provided resources. You might consider inviting a few students to share their reflections on the varying displays.
+
+We recommend printing and distributing the student-facing @handout{beautiful-data-rubric.adoc} to help students understand the scope of the project and your expectations at the outset. Teachers are welcome and encouraged to edit and adapt the rubric for their unique classroom context.}
+
+@lesson-instruction{
+On @opt-printable-exercise{beautiful-data-reflect.adoc}, respond to the first two questions.}
+
+Today, you're going to get a chance to create your own customized data visualization - one that connects artistic expression to data in ways that are relevant and meaningful.
+
+The first step is to find your data!
+
+@lesson-instruction{
+- @lesson-link{choosing-your-dataset, Choose a dataset} that interests you - or one that you're already working with - and use `bar-chart`, `pie-chart`, `histogram`, or `scatter-plot` to create a simple, interesting display.
+- Write down a title for the display you created on @opt-printable-exercise{beautiful-data-reflect.adoc}, Q3.
+- Respond to Q4: What does this display show? Why would it be interesting to someone else?
+}
+
+=== Investigate
+
+@right{@image{images/age-v-weeks.gif, 300}}
+Once students have created a basic display, things start to get interesting!
+
+The scatter plot on the right suggests a moderate correlation between age and weeks: older animals take longer to be adopted, but it’s not as direct a connection as others might think! It's time to get curious, and ask ourselves: *What data is missing?*
+
+In the above scatter plot, we might wonder whether the species matters, how big the animal is, or whether or not it’s fixed. This gives us both _an observation_ and a _more advanced question_ to ask about our data: *Different species of animals have very different lifespans. Is the correlation between age and adoption time different for each species?*
+
+@lesson-instruction{
+- Think about at least two columns that might give you more insight into your data, and why those other columns matter in the display(s) you’ve chosen.
+- Write the observations and questions in @opt-printable-exercise{beautiful-data-reflect.adoc}.}
```

Comment:
Add question numbers here (and anywhere else you explicitly want them to do PART of this page)